pootle/static/js/browser/components/Stats.js

```diff
@@ -45,12 +47,19 @@ const Stats = React.createClass({
   },
 
   render() {
+    if (!this.state.topContributors.length) {
```

Comment:
The behavior is in no way different from the current code, which checks for `top_scorers` in the template.